src/main/java/com/springweb/dv_spring_web_mongo/service/ProjectService.java

```diff
@@ -30,21 +27,30 @@
     }
 
     public ProjectDTO getProjectById(String id) {
-        Optional<Project> project = projectRepository.findById(id);
-        return project.get().convertToDTO();
+        getById(id);
+        return projectRepository.findById(id).get().convertToDTO();
```

Comment:
Why you cannot use `getById(id)` instead of `projectRepository.findById(id).get()`?